src/main/java/com/simleetag/homework/api/domain/work/CategoryService.java

```diff
@@ -86,11 +86,11 @@ public List<TaskRateResponse> calculateTaskRatesByDueDates(Long memberId, LocalD
         Period period = Period.between(startDate, endDate);
         List<TaskRateResponse> list = new ArrayList<>();
         for (int i = 0; i < period.getDays() + 1; i++) {
-            long allTasks = taskDslRepository.findAllWithTaskGroupByHomeIdAndOwnerAndDueDate(memberId, startDate.plusDays(i)).size();
-            long doneTasks = taskDslRepository.findAllWithTaskGroupByHomeIdAndOwnerAndDueDate(memberId, startDate.plusDays(i)).stream().filter(task -> task.getTaskStatus().equals(TaskStatus.COMPLETED)).count();
-            long rate = 0;
+            double allTasks = taskDslRepository.findAllWithTaskGroupByHomeIdAndOwnerAndDueDate(memberId, startDate.plusDays(i)).size();
+            double doneTasks = taskDslRepository.findAllWithTaskGroupByHomeIdAndOwnerAndDueDate(memberId, startDate.plusDays(i)).stream().filter(task -> task.getTaskStatus().equals(TaskStatus.COMPLETED)).count();
```

Comment:
사소하지만 같은 쿼리를 2번 날리네요!

```suggestion
var tasks = taskDslRepository.findAllWithTaskGroupByHomeIdAndOwnerAndDueDate(memberId, startDate.plusDays(i));
double allTasks = tasks.size();
double doneTasks = tasks.stream().filter(task -> task.getTaskStatus().equals(TaskStatus.COMPLETED)).count();
```

Reply:
감사합니다! 해당 부분 반영했습니다~